Gruntfile.js

```diff
@@ -23,6 +35,60 @@ module.exports = function(grunt) {
             });
     });
 
+    grunt.registerTask("hugo-server", function() {
+        const done = this.async();
+
+        grunt.log.writeln("Running Hugo server");
+        grunt.util.spawn({
+            cmd: "hugo",
+            args: ["server"],
+            opts: {stdio: 'inherit'}
+        },
+            function(error, result, code) {
+                if (code == 0) {
+                    grunt.log.ok("Thanks for using Hugo!");
+                } else {
+                    grunt.fail.fatal(error);
+                }
+                done();
+            });
+    });
+
+    grunt.registerTask("hugo-server-fullrender", function() {
+        const done = this.async();
+
+        grunt.log.writeln("Running Hugo server with Fast Render disabled");
+        grunt.util.spawn({
+            cmd: "hugo",
+            args: ["server", "--disableFastRender"],
+            opts: {stdio: 'inherit'}
+        },
+            function(error, result, code) {
+                if (code == 0) {
+                    grunt.log.ok("Thanks for using Hugo!");
+                } else {
+                    grunt.fail.fatal(error);
+                }
+                done();
+            });
+    });
+
+    grunt.registerTask("print-hugo-version", function() {
+        const done = this.async();
+        grunt.util.spawn({
+            cmd: "hugo",
+            args: ["version"],
+            opts: {stdio: 'inherit'}
+        },
+            function(error, result, code) {
```

Comment:
This appears to be tabbed over once too many times.

Reply:
On a related note.. at some point we should add `prettifier` and an editor config to the project. 👍 Enforce consistent code automagically.